app/policies/project_policy.rb

```diff
@@ -16,12 +16,18 @@ def join?
     true
   end
 
+  def archive?
+    !record.archived && import?
```

Comment:
why `import?` ?

Reply:
On the previous versions of the policies file the archive policies copied from from the import one, and the unarchive one copied from the archive like this:
`  alias archived? update?
  alias import_upload? import?
  alias archive? import?
  alias unarchive? archive?
  alias destroy? archive?
  alias share? archive?
  alias unshare? share?
  alias transfer? share?
  alias ownership? share?`

So in order to retain this behaviour I did this.